api/src/opentrons/hardware_control/api.py

```diff
@@ -591,9 +591,9 @@ def halt(self):
         asyncio.run_coroutine_threadsafe(self._execution_manager.cancel(),
                                          self._loop)
 
-    async def stop(self):
+    async def stop(self, home_after: bool = False):
```

Comment:
Oops, wrong default!

```suggestion
    async def stop(self, home_after: bool = True):
```